src/util.js

```diff
@@ -425,12 +417,12 @@ const util = {
   },
 
   getHardwareConcurrency: function() {
-    if (util.detectNode()) {
-      const os = require('os');
-      return os.cpus().length;
+    if (typeof navigator !== 'undefined') {
+      return navigator.hardwareConcurrency || 1;
     }
 
-    return (typeof navigator !== 'undefined' && navigator.hardwareConcurrency) || 1;
+    const os = require('os'); // if node
```

Comment:
If we're gonna write a comment let's expand it a bit, e.g.
```suggestion
    const os = require('os'); // Assume we're on Node.js.
```